src/App.tsx

```diff
@@ -47,11 +53,75 @@ const watchOffers = async (wallet: any, setOffers: (offers: any) => void) => {
   }
 };
 
+const watchGovernance = async (
+  leader: any,
+  unserializer: any,
```

Comment:
while `makeFollower` takes this in its `unserialized` property, it's really a `marshal: Marshal` object.

Endo exports types so you can import its Marshal. Even better, define a type locally that fills its generic slot and indicate how it marshals (just repeats the slot key):
```
import type { Marshal } from '@endo/marshal';

type IdentityMarshal = Marshal<string>;
```
Then this would be,
```suggestion
  marshal: IdentityMarshal,
```

All this should be in `.ts` files not `.tsx` because it's independent of the UI state.

Reply:
Done.